acados/utils/math.c

```diff
@@ -1092,41 +1093,6 @@ void acados_eigen_decomposition(int dim, double *A, double *V, double *d, double
 
 
 
-void eigen_decomposition(int dim, double *A, double *V, double *d)
```

Comment:
are you sure this is not used in the regularization???

Reply:
Yes. I found that all calls were replaced with calls to `acados_eigen_decomposition` which doesnt use `calloc`.